src/content/docs/extras.mdx

```diff
@@ -320,6 +320,213 @@ str_detect("Hello world", "ello")
+ 2  2014     1     1        -5       -17 AA      MIA        161     1085    16 EWR   
+ 3  2014     1     1       191       185 AA      DFW        214     1372    16 EWR   
+ 4  2014     1     1        -1        -2 AA      DFW        214     1372    14 EWR   
+ 5  2014     1     1        -3       -10 AA      MIA        154     1085     6 EWR   
+ 6  2014     1     1         4       -17 AA      DFW        215     1372     9 EWR   
+ 7  2014     1     1        -3        27 AA      DFW        211     1372     7 EWR   
+ 8  2014     1     1        -5        -8 AA      MIA        157     1085    11 EWR   
+ 9  2014     1     1        -9       -16 AA      DFW        218     1372    19 EWR   
+10  2014     1     1         4       -12 AS      SEA        347     2402    18 EWR   
+# ℹ more rows
+# ℹ Use `print(n = ...)` to see more rows
+```
+
+With the data transfered to a duckdb database, we can continue as before with dplyr and `collect()` syntax. Under the hood, the duckdb computation engine will be used which is has a more complete feature set allowing more operations (e.g. rolling windows) to be used. One nice thing `duckdb`/`arrow` do is the `explain()` function which explains what is happening under the hood for those curious. 
+
+```r
+query = dat_duckdb |>
+  filter(origin == "JFK") |>
+  group_by(month) |>
+  summarize(
+    mean_dep_delay = mean(dep_delay, na.rm = TRUE), 
+    n_flights = n()
+  )
+explain(query)
+```
+```text
+<SQL>
+SELECT "month", AVG(dep_delay) AS mean_dep_delay, COUNT(*) AS n_flights
+FROM arrow_001
+WHERE (origin = 'JFK')
+GROUP BY "month"
+
+<PLAN>
+physical_plan
+┌───────────────────────────┐
+│       HASH_GROUP_BY       │
+│   ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─   │
+│             #0            │
+│          avg(#1)          │
+│        count_star()       │
+└─────────────┬─────────────┘                             
+┌─────────────┴─────────────┐
+│         PROJECTION        │
+│   ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─   │
+│           month           │
+│         dep_delay         │
+└─────────────┬─────────────┘                             
+┌─────────────┴─────────────┐
+│        ARROW_SCAN         │
+│   ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─   │
+│           month           │
+│         dep_delay         │
+│   ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─   │
+│  Filters: origin=JFK AND  │
+│     origin IS NOT NULL    │
+│   ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─   │
+│           EC: 0           │
+└───────────────────────────┘
+```
+
+Then, the data can be `collect()`ed into a data.frame:
+```r
+collect(query)
+```
+```text
+# A tibble: 10 × 3
+   month mean_dep_delay n_flights
+   <int>          <dbl>     <dbl>
+ 1    10           6.97      8605
+ 2     8           8.66      8986
+ 3     9           3.76      8228
+ 4     3           7.17      8253
+ 5     4           7.00      8070
+ 6     5          10.9       8305
+ 7     2          19.2       6779
+ 8     6           9.81      8422
+ 9     7          17.6       8730
+10     1          27.0       7105
+```
+
```

Comment:
I would add a P.S., saying that pivot / reshape functionality is also supported on the DuckDB backend, but the **tidyr** package should be loaded alongside dplyr for this too work.

Reply:
Reshaping is an example of something that DuckDB can do, but acero can't. 

(There are a bunch more, including some of my own personal favourite use cases: spatial data manipulation. DuckDB has a fully function and very fast spatial spatial computation set. See also the [duckdbfs](https://cboettig.github.io/duckdbfs/#spatial-data) package which helps to integrate some stuff behind the scenes.)